test/integration/compAnchorIntegration.js

```diff
@@ -0,0 +1,559 @@
+    // create resource ID using anchor address
+    resourceID = helpers.createResourceID(OriginChainLinkableAnchorInstance.address, 0);
+    initialResourceIDs = [resourceID];
+    originInitialContractAddresses = [DestChainLinkableAnchorInstance.address];
+    destInitialContractAddresses = [OriginChainLinkableAnchorInstance.address];
+    // initialize anchorHanders 
+    await Promise.all([
+      AnchorHandlerContract.new(OriginBridgeInstance.address, initialResourceIDs, originInitialContractAddresses)
+        .then(instance => OriginAnchorHandlerInstance = instance),
+      AnchorHandlerContract.new(DestBridgeInstance.address, initialResourceIDs, destInitialContractAddresses)
+        .then(instance => DestAnchorHandlerInstance = instance),
+    ]);
+    // increase allowance and set resources for bridge
+    await Promise.all([
+      OriginBridgeInstance.adminSetResource(OriginAnchorHandlerInstance.address, resourceID, OriginChainLinkableAnchorInstance.address),
+      DestBridgeInstance.adminSetResource(DestAnchorHandlerInstance.address, resourceID, DestChainLinkableAnchorInstance.address)
+    ]);
+     // set bridge and handler permissions for anchors
+    await Promise.all([
+      OriginChainLinkableAnchorInstance.setHandler(OriginAnchorHandlerInstance.address, {from: sender}),
+      OriginChainLinkableAnchorInstance.setBridge(OriginBridgeInstance.address, {from: sender}),
+      DestChainLinkableAnchorInstance.setHandler(DestAnchorHandlerInstance.address, {from: sender}),
+      DestChainLinkableAnchorInstance.setBridge(DestBridgeInstance.address, {from: sender})
+    ]);
+
+    createWitness = async (data) => {
+      const wtns = {type: 'mem'};
+      await snarkjs.wtns.calculate(data, path.join(
+        'test',
+        'fixtures',
+        'poseidon_bridge_2.wasm'
+      ), wtns);
+      return wtns;
+    }
+
+    tree = new MerkleTree(merkleTreeHeight, null, prefix)
+    zkey_final = fs.readFileSync('test/fixtures/circuit_final.zkey').buffer;
+  });
+
+  it('[sanity] bridges configured with threshold and relayers', async () => {
+    assert.equal(await OriginBridgeInstance._chainID(), originChainID);
+    assert.equal(await OriginBridgeInstance._relayerThreshold(), relayerThreshold)
+    assert.equal((await OriginBridgeInstance._totalRelayers()).toString(), '1')
+    assert.equal(await DestBridgeInstance._chainID(), destChainID)
+    assert.equal(await DestBridgeInstance._relayerThreshold(), relayerThreshold)
+    assert.equal((await DestBridgeInstance._totalRelayers()).toString(), '1')
+  })
+  
+  it('Wrapping should fail if Anchor does not have MINTER_ROLE', async () => {
+    //originToken is voted to be approved for token wrapper on origin chain
+    await helpers.addTokenToWrapper(originGov, originWrapperToken, originToken, bravoAdmin, states);
+    // revoke anchor permissions
+    await originWrapperToken.revokeRole(MINTER_ROLE, OriginChainLinkableAnchorInstance.address, {from: sender});
```

Comment:
Can you describe this more? Is the anchor the minter and if so why can the sender revoke that role? What role is the sender?